### PR TITLE
create styled template for main Activity screen with no data

### PR DESCRIPTION
**Pull request by elim322**
- Created a template with placeholders for images that will come from API. 
- Still waiting for fonts from designers therefore there are no fonts enabled. 
- Image background only works with percentages.

### CODE REVIEW

File: js/screens/Activity/ActivityContainer.js [hunk 1]
```diff
@@ -3,11 +3,13 @@ import Activity from "./Activity";
 import { View } from "react-native";
```
Comment (siddharthparmar7) [inline js/screens/Activity/ActivityContainer.js]: remove unnecessary imports.

File: js/screens/Activity/styles.js [hunk 2]
```diff
@@ -1,2 +1,34 @@
-import { StyleSheet } from "react-native";
-export default (style = StyleSheet.create({}));
+import { StyleSheet, Dimensions } from "react-native";
+const styles = StyleSheet.create({
+  background: {
+    width: "100%",
+    height: "100%"
```
Comment (siddharthparmar7) [inline js/screens/Activity/styles.js]: no %. Use Dimensions or flexbox or resize mode.

File: js/screens/Activity/styles.js [hunk 3]
```diff
@@ -1,2 +1,34 @@
...
+  },
+  container: {
+    flex: 1,
+    marginTop: 100
```
Comment (siddharthparmar7) [inline js/screens/Activity/styles.js]: margin 100? remove it. Use Flexbox.

File: js/screens/Activity/styles.js [hunk 4]
```diff
@@ -1,2 +1,34 @@
...
+  },
+  pickActivity: {
+    alignSelf: "center",
+    marginBottom: 125
```
Comment (siddharthparmar7) [inline js/screens/Activity/styles.js]: same.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Move all of your images into an images dir in your assets and no image should hang out by itself.

For the Activity screen, your margin/padding for the title of each activity seems off. Please take a look at the design and fix it.

Also, background for the activity screen is different in the design than your code. And each Activity has different width in design than your code. Please fix it.

Make the requested changes. Thank you.

**Review by 10thfloor — APPROVED** [review]

**Pull request merged**